### PR TITLE
hw-fruitshop

### PR DESCRIPTION
**Pull request by LizaBalshv**

### CODE REVIEW

**Comment by LizaBalshv** [comment]
don't understand what the mistakes are...

File: src/main/java/core/basesyntax/service/Parser.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Parser {
+    FruitTransaction parse(String data);
+
+    List<FruitTransaction> convertFruitDataToTransactions(List<String> fruitData);
```
Comment (Vasul-Pankiv) [inline src/main/java/core/basesyntax/service/Parser.java:9]:
```suggestion
    List<FruitTransaction> parseAll(List<String> fruitData);
```

**Review by Vasul-Pankiv — COMMENTED** [review]
Great job!!! Left some suggestions for better naming

**Review by Vasul-Pankiv — CHANGES_REQUESTED** [review]
Great job!!! Left some suggestions for better naming

**Review by olekskov — APPROVED** [review]
Good job

**Pull request closed without merging**